app/views/application/index.html.erb

```diff
@@ -0,0 +1,6 @@
+<h1>Welcome to Farmer's Market Central!</h1>
+
+<div class="mainlinks">
+  <%= link_to "Markets", markets_path %>
+  <%= link_to "Vendors", vendors_path %>
+</div>
```

Comment:
As a user experience thing, it might be more welcoming to the user to make these links more obvious on the home page.  Maybe larger? Some instructions? Or a css hover effect on the links so that it is more clear what they are.

I love your layout and colors on this page!